src/App.tsx

```diff
@@ -174,6 +174,7 @@ function App() {
     if (shouldTriggerSafetyCheck()) {
       dispatch(info("Safety Check: Always verify you're on app.olympusdao.finance!"));
     }
+    loadDetails("app");
```

Comment:
probably time to start looking again at killing app slice completely, but for now will reinstate it.

Reply:
whoops. Good catch. Yeah, i think the last element using it is migration modal